docs/DeveloperGuide.adoc

```diff
@@ -13,7 +13,24 @@ endif::[]
 ifdef::env-github,env-browser[:outfilesuffix: .adoc]
 :repoURL: https://github.com/se-edu/addressbook-level4/tree/master
 
-By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`
+By: `Team W13-B1`      Since: `AUG 2017`      Licence: `MIT`
+
+== Introduction
+
+=== Purpose
+
+UniCity is a contacts management desktop application implemented in Java. This developer guide aims to provide
```

Comment:
I think it should be "contact management"

Reply:
Alright fixed!